graphix/pattern.py

```diff
@@ -1212,6 +1243,40 @@ def perform_pauli_measurements(self, leave_input=False):
         """
         measure_pauli(self, leave_input, copy=False)
 
+    def view_graph(self, figsize=None, pauli_indicator=True, local_clifford_indicator=False, save=False, filename=None):
+        """Visualize the underlying graph of the pattern with flow or gflow structure.
+
+        Parameters
+        ----------
+        figsize : tuple
+            Figure size of the plot.
+        pauli_indicator : bool
+            If True, the nodes are colored according to the measurement angles.
+        local_clifford_indicator : bool
+            If True, indexes of the local Clifford operator are displayed adjacent to the nodes.
+        save : bool
+            If True, the plot is saved as a png file.
+        filename : str
+            Filename of the saved plot.
+        """
+
+        nodes, edges = self.get_graph()
+        g = nx.Graph()
+        g.add_nodes_from(nodes)
+        g.add_edges_from(edges)
+        vin = self.input_nodes if self.input_nodes is not None else []
+        vout = self.output_nodes
+        vis = GraphVisualizer(g, vin, vout)
+        if pauli_indicator:
+            angles = self.get_angles()
+        else:
+            angles = None
+        if local_clifford_indicator:
+            local_clifford = self.get_local_clifford()
+        else:
+            local_clifford = None
+        vis.visualize(figsize=figsize, angles=angles, local_clifford=local_clifford, save=save, filename=filename)
+
```

Comment:
this should be named `visualize_flow` instead, perhaps?

Reply:
This method also visualize gflow graph or no flow/gflow graph, so a name with a broader meaning is preferred.
I set the name `draw_graph()`